packages/platform/platform-response-filter/src/services/PlatformResponseFilter.ts

```
@@ -15,57 +18,82 @@ export class PlatformResponseFilter {
+   * @param $ctx The context.
    */
-  transform(data: unknown, ctx: BaseContext) {
-    const {response} = ctx;
+  async transform(data: unknown, $ctx: BaseContext): Promise<unknown> {
+    const {endpoint} = $ctx;
 
-    if (ctx.endpoint?.operation) {
-      const bestContentType = this.contentTypeResolver(data, ctx);
+    if (endpoint) {
+      if (endpoint.view) {
+        data = await this.renderView(data, $ctx);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)